src/reference/asciidoc/aggregator.adoc

```diff
@@ -622,76 +558,89 @@ public String[] processRelease(List<Message<String>> messages){
+Starting with version 4.0, two new mutually exclusive attributes have been introduced: `group-timeout` and `group-timeout-expression` (see the earlier description).
+// TODO This needs a link, but I don't see a target for it.
+In some cases, you may need to emit the aggregator result (or discard the group) after a timeout if the `ReleaseStrategy` does not release when the current message arrives.
+For this purpose, the `groupTimeout` option lets scheduling the `MessageGroup` be forced to complete, as the following example shows:
+
+====
 [source,xml]
 ----
 <aggregator input-channel="input" output-channel="output"
         send-partial-result-on-expiry="true"
         group-timeout-expression="size() ge 2 ? 10000 : -1"
         release-strategy-expression="messages[0].headers.sequenceNumber == messages[0].headers.sequenceSize"/>
 ----
+====
 
-With this example, the normal _release_ will be possible if the aggregator receives the last message in sequence as defined by the `release-strategy-expression`.
-If that specific message does not arrive, the `groupTimeout` will force the group complete after 10 seconds as long as the group contains at least 2 Messages.
+With this example, the normal release is possible if the aggregator receives the last message in sequence as defined by the `release-strategy-expression`.
+If that specific message does not arrive, the `groupTimeout` forces the group to complete after ten seconds, as long as the group contains at least two Messages.
 
-The results of forcing the group complete depends on the `ReleaseStrategy` and the `send-partial-result-on-expiry`.
-First, the release strategy is again consulted to see if a _normal_ release is to be made - while the group won't have changed, the `ReleaseStrategy` can decide to release the group at this time.
-If the release strategy still does not release the group, it will be expired.
-If `send-partial-result-on-expiry` is `true`, existing messages in the (partial) `MessageGroup` will be released as a normal aggregator reply Message to the `output-channel`, otherwise it will be discarded.
+The results of forcing the group to complete depends on the `ReleaseStrategy` and the `send-partial-result-on-expiry`.
+First, the release strategy is again consulted to see if a normal release is to be made.
+While the group has not changed, the `ReleaseStrategy` can decide to release the group at this time.
+If the release strategy still does not release the group, it is expired.
+If `send-partial-result-on-expiry` is `true`, existing messages in the (partial) `MessageGroup` are released as a normal aggregator reply message to the `output-channel`.
+Otherwise, it is discarded.
 
-There is a difference between `groupTimeout` behavior and `MessageGroupStoreReaper` (see <<aggregator-config>>).
+There is a difference between `groupTimeout` behavior and `MessageGroupStoreReaper` (see "`<<aggregator>>`").
```

Comment:
This points to fully different paragraph unlike the old one `aggregator-config`.
I think this indeed was the point to link end-user to the particular paragraph, but not the whole chapter